docs/admin/authorization/abac.md

```diff
@@ -24,7 +24,7 @@ properties:
 
   - Versioning properties:
     - `apiVersion`, type string; valid values are "abac.authorization.kubernetes.io/v1beta1". Allows versioning and conversion of the policy format.
-    - `kind`, type string: valid values are "Policy". Allows versioning and conversion of the policy format.
+    - `kind`, type string; valid values are "Policy". Allows versioning and conversion of the policy format.
```

Comment:
I think I had look into the formatting of my changes, and could you tell me which bullet is not showing up correctly, the ";" or "."?